docs/deploy.md

```diff
@@ -236,3 +237,281 @@ The target mainchain address of a backward transfer has to be a valid address: s
 
 ```
 
+## Deploying a Contract using Hardhat
+To deploy a contract, you will use a deployment script that is specific for your smart contract. 
+Hardhat script.  Go to the script/ folder and find the following deploy.ts file:
+
+### Deploy contract ERC20.sol
+To deploy your contract ERC20.sol, create a folder in your Hardhat project and name it, **scripts/**. Copy this script (below) and name it **ERC20.ts**.  
+
+Before you can deploy a contract, you need to add the import statement by running the command:
+
+```
+import {ether} from hardhat;
+```
+
+
+Use the deployment script for the contract, ERC20.sol:  
+```
+const deployERC20 = async () => {
+const ERC20 = await ethers.getContractFactory("ERC20"); const erc20 = await ERC20.deploy("Test", "TST");
+await erc20.deployed();
+console.log(erc20.address); 
+};
+deployERC20();
+```
+Run the command to deploy the contract:
+
+```
+npx hardhat run ./scripts/ERC20.ts
+```
+You will see the following execution results:
+
+```
+0x82534fBC27671eEd1F99d342CAd6be213f541a87
+```
+
+### Deploy contract TestPayable.sol
```

Comment:
Just out of curiosity, why do we give 2 example contracts in this section?

Reply:
The SME wanted to show payable contract, other than that, they seem to work.